csrc/ir_nodes.cpp

```diff
@@ -2478,10 +2478,19 @@ IterDomain* IterDomain::resize(
         right_expansion);
   }
 
+  auto iter_type = in->getIterType();
+  auto is_symbolic = !resized_id_size->isConstInt();
+  if (symbolic) {
+    iter_type = IterType::Symbolic;
+  } else {
+    auto extent_val = resized_id_size->getInt().value();
+    iter_type = extent_val == 1 ? IterType::Broadcast : IterType::Iteration;
+  }
```

Comment:
@naoyam should we set to Iteration by default here or to the same IterType as the input? If it is ScatterGather,  but resized to size 1, I'm not sure what it should be set to here.

Reply:
Ignore `ScatterGather` for now. I believe we can remove it, and it's not used for `take_along_axis`. I'll remove it from `torch_gather` as well.